tests/transformers/models/test_causal_lm_models.py

```diff
@@ -122,7 +163,12 @@ def test_causal_lm_pytorch_vs_kv_vs_ort_vs_ai100(model_name):
     pytorch_hf_tokens = np.vstack(pytorch_hf_tokens)
 
     qeff_model = QEFFAutoModelForCausalLM(model_hf, f"{model_name}")
-    onnx_model_path = qeff_model.export()
+    _, onnx_model_path = qualcomm_efficient_converter(
+        model_name=model_name,
+        model_kv=qeff_model,
+        tokenizer=tokenizer,
+        full_batch_size=4,
+    )
```

Comment:
need to test fbs and without fbs separately

Reply:
We can update this in a different PR, as it is not related.